corelib/Scarb.lock

```diff
@@ -0,0 +1,6 @@
+# Code generated by scarb DO NOT EDIT.
+version = 1
+
+[[package]]
+name = "core"
+version = "2.2.0"
```

Comment:
Is this the right version of the corelib? Scarb is `2.3.0` so perhaps you need to update the local corelib content (do we even still need it?)

Reply:
Aah demn, forgot about the corelib 😂